plugins/nitrokey/meson.build

```diff
@@ -0,0 +1,43 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginNitrokey"']
+
+shared_module('fu_plugin_nitrokey',
+  sources : [
+    'fu-device-nitrokey.c',
+    'fu-nitrokey-common.c',
+    'fu-plugin-nitrokey.c',
+  ],
+  include_directories : [
+    include_directories('../..'),
+    include_directories('../../src'),
+    include_directories('../../libfwupd'),
+  ],
+  install : true,
+  install_dir: plugin_dir,
+  c_args : cargs,
+  dependencies : [
+    plugin_deps,
+  ],
+)
+
+if get_option('enable-tests')
+  e = executable(
+    'nitrokey-self-test',
+    sources : [
+      'fu-nitrokey-common.c',
+      'fu-self-test.c',
+    ],
+    include_directories : [
+      include_directories('../..'),
+      include_directories('../../src'),
+      include_directories('../../libfwupd'),
+    ],
+    dependencies : [
+      plugin_deps,
+      valgrind,
+    ],
+    link_with : [
+      libfwupdprivate,
+    ],
+  )
+  test('raspberrypi-self-test', e)
```

Comment:
whoops, this should probably be `nitrokey-self-test` for the binary name

Reply:
Fixed, thanks.